fluxio_parser/visitors/script.py

```diff
@@ -3,12 +3,12 @@
 from collections import defaultdict
 from typing import Any, Dict, Set
 
-from ..exceptions import assert_supported_operation, UnsupportedOperation
-from ..resource_decorators import RESOURCE_DECORATOR_MAP
-from ..util import parse_options
-from .event_processor import EventProcessorVisitor
-from .state_machine import StateMachineVisitor
-from .task import TaskVisitor
+from ..exceptions import assert_supported_operation, UnsupportedOperation  # noqa
+from ..resource_decorators import RESOURCE_DECORATOR_MAP  # noqa
+from ..util import parse_options  # noqa
+from .event_processor import EventProcessorVisitor  # noqa
+from .state_machine import StateMachineVisitor  # noqa
+from .task import TaskVisitor  # noqa
```

Comment:
does the linter not like `..`? Also any reason you use that over the fully qualified path?

Reply:
The linter is ok with `..` but the sort order plugin wants to put `.` before `..`. Reason for relative links is mostly a personal preference. I might do a follow up and replace everything w absolute.